sycl/doc/PreprocessorMacros.md

```diff
@@ -57,6 +57,11 @@ This file describes macros that have effect on SYCL compiler and run-time.
 
 ## Version macros
 
+- `__LIBSYCL_TIMESTAMP` is set to an integer literal which corresponds to a date
+  when the latest commit into `sycl/` subproject was done. It could be used
```

Comment:
> - `__LIBSYCL_TIMESTAMP` is set to an integer literal which corresponds to a date
  when the latest commit into `sycl/` subproject was done.

I'm not sure that is what the code in sycl/cmake/modules/SYCLVersionFromVCS.cmake does. According to my understanding, it infers the date of the latest commit to the whole repository. To get the commit modifying files in `sycl/` directory, the command must include `-- ${SYCL_ROOT_SOURCE_DIR}`.

Reply:
That was an oversight on my part, by some reason I thought that `WORKING_DIRECTORY` would be enough. Fixed in f5418d7da2ef17ffebf5d53d0bdc193e522186a8